src/console-services.yml

```diff
@@ -159,22 +303,90 @@
   description: 'Lambda - Run Code without Thinking about Servers'
   url: 'https://console.aws.amazon.com/lambda/home'
 
-- command: machinelearning
-  description: 'Machine Learning - Build Smart Applications Quickly and Easily'
-  url: 'https://console.aws.amazon.com/machinelearning/home'
+- command: lex
+  description: 'Lex - Build Voice and Text Chatbots'
+  url: 'https://console.aws.amazon.com/lex/home'
+
+- command: licensemanager
+  description: 'License Manager - Set rules to manage, discover, and report third-party license usage proactively'
+  url: 'https://console.aws.amazon.com/license-manager/home'
+
+- command: lightsail
+  description: 'Lightsail - Launch and Manage Virtual Private Servers'
+  url: 'https://lightsail.aws.amazon.com/ls/webapp/home'
+
+- command: macie
+  description: 'Macie - Amazon Macie classifies and secures your business-critical content.'
+  url: 'https://macie.aws.amazon.com/'
```

Comment:
this isn't a valid URL. i'd revert this URL to the previous one

Reply:
Fixed